src/SWP/Bundle/WebRendererBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Web Renderer Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\WebRendererBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files.
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html#cookbook-bundles-extension-config-class}
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $treeBuilder->root('swp_web_renderer')
+            ->children()
+                ->arrayNode('device_listener')
+                    ->canBeEnabled()
+                    ->info('Enable device detection in templates loader')
+                    ->children()
+                        ->scalarNode('enabled')->defaultFalse()->end()
```

Comment:
you dont need to specify this if you already used `canBeEnabled` - http://symfony.com/doc/current/components/config/definition.html#optional-sections